packages/flutter_tools/test/general.shard/unified_analytics_test.dart

```diff
@@ -30,14 +32,25 @@ void main() {
     );
   });
 
+  group('Unit testing util:', () {
+    testWithoutContext('getEnabledFeatures', () {
```

Comment:
I should probably update this test so that it can handle a return type of `cli-animations,enable-flutter-preview` and `enable-flutter-preview,cli-animations`... the order right now is determined by the order of `allFeatures`